java/ql/integration-tests/linux-only/kotlin/custom_plugin/test.py

```diff
@@ -1,6 +1,25 @@
 from create_database_utils import *
-import subprocess
 
-subprocess.call("./build_plugin", shell=True)
+import pathlib
+import shutil
+
+this_dir = pathlib.Path(__file__).resolve().parent
+cwd = pathlib.Path.cwd()
+builddir = cwd / 'build'
+
+builddir.mkdir(exist_ok=True)
+
+try:
+    runSuccessfully(['bazel', f'--output_user_root={builddir}', '--max_idle_secs=1', 'build',
```

Comment:
Is this using `tools/bazel`? If not, should it be?

Reply:
it definitely will on github actions (where `bazel` is `bazelisk`) and on most local setups (where `tools/bazel` is the recommended `bazel`, and even if going for the upstream recommended installation which is via bazelisk we still end up picking that up.

That said, we can remove the unknown about special local setups by calling into `tools/bazel`. I need to look how to pick up the semmle-code root in integration tests though

Reply:
ah, wait x2 :sweat_smile: I do pass `cwd=this_dir`, which will indeed make it run from within a semmle-code checkout. Still, better to be sure about it and call `tools/bazel`.